README.md

```diff
@@ -21,6 +21,29 @@ Or install it yourself as:
 $ gem install perilune
 ```
 
+## Configuration
+You can use diffrent queue name for cron job:
+
+```ruby
+Periline::Configuration do |config|
+  config.queue_name = 'your_queue_name'
+end
+```
+
+> Note - Perilune is using *default* as queue name.
+
+You can configure the driver for Trifle::Stats :
+
+```ruby
+Periline::Configuration do |config|
+  config.trifle_stat_driver = ::Trifle::Stats::Driver::Redis.new
+  config.trifle_stat_track_ranges = %i[hour day week month quarter year]
+  config.trifle_stat_time_zone = 'GMT'
+  config.trifle_stat_beginning_of_week = :monday
```

Comment:
These dont need to be configurable. It may end up more confusing to users. For example in dahsboard we wanna add, we will need hourly stats, what if user configures only monthly stats? Etc. If they will ask for this to be configurable, we can always add this later.

Reply:
If you let user override the `stats_driver_config` method, you allow them to override all configuration that will break stats for dashboard.